crates/svm-cli/src/cli.rs

```diff
@@ -13,21 +12,21 @@ pub fn new_app<'a, 'b>() -> App<'a, 'b> {
                 .about("Encode various transaction types")
                 .setting(AppSettings::SubcommandRequiredElseHelp)
                 .subcommand(
-                    App::new("app-template")
```

Comment:
I personally love using `_` instead of `_` but it's OK with both.

Reply:
I didn't understand what you mean.. you mentioned the same char twice.